MAUI/Scheduler/load-on-demand.md

```diff
@@ -0,0 +1,311 @@
+            new SolidColorBrush(Color.FromArgb("#FF0F8644")),
+            new SolidColorBrush(Color.FromArgb("#FF01A1EF"))
+        };
+
+        var subjectCollection = new ObservableCollection<string>
+        {
+            "Business Meeting",
+            "Conference",
+            "Medical check up",
+            "Performance Check",
+            "Consulting",
+            "Project Status Discussion",
+            "Client Meeting",
+            "General Meeting",
+            "Yoga Therapy",
+            "GoToMeeting",
+            "Plan Execution",
+            "Project Plan"
+        };
+
+        Random ran = new();
+        int daysCount = visibleDates.Count;
+        DateTime visibleStartDate = visibleDates.FirstOrDefault();
+        var appointments = new ObservableCollection<SchedulerAppointment>();
+        for (int i = 0; i < 10; i++)
+        {
+            var startTime = visibleStartDate.AddDays(ran.Next(0, daysCount + 1)).AddHours(ran.Next(9, 16));
+            appointments.Add(new SchedulerAppointment()
+            {
+                StartTime = startTime,
+                EndTime = startTime.AddHours(1),
+                Subject = subjectCollection[ran.Next(0, subjectCollection.Count)],
+                Background = brush[ran.Next(0, brush.Count)]
+            });
+        }
+
+        return appointments;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N> The `QueryAppointmentsCommand` will be raised if any one of the following action is performed,
+* Once the `ViewChanged` event is raised, the `QueryAppointmentsCommand` will be raised.
+* If an appointment has been added or removed in the current time visible date, then the `QueryAppointments` command will not be triggered. Because the appointments for that visible date are already loaded.
+* In the scheduler agenda view, the `QueryAppointments` event is used to load more appointments when the new month is loaded on view, whereas the new month appointments are added in scheduler `AppointmentsSource.`
+* Other than agenda view, the scheduler `AppointmentsSource` can be reset for a new visible date range to improve appointment loading performance.
+
+## Show busy indicator
+
+The `Scheduler` supports to show the busy indicator view by using the `ShowBusyIndicator` property of the `SfScheduler.` The default value is set to `false.` If the value is set to `true` and visible dates are changed then the busy indicator will be loaded on view.
+
+{% tabs %}
+{% highlight xaml %}
+
+ <scheduler:SfScheduler x:Name="Scheduler" 
+                        View="Week"
+                        ShowBusyIndicator="True">
+ </scheduler:SfScheduler>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.Scheduler.View = SchedulerView.Week;
+this.Scheduler.ShowBusyIndicator = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+N>  
+* You might start and stop the animation before and after the appointments loaded, when using the `QueryAppointments,` or `QueryAppointmentsCommand.` 
```

Comment:
When using the `QueryAppointments` or `QueryAppointmentsCommand,` you can start and stop the animation before and after the appointments load.